<?xml version="1.0"?>

<launch>
    <arg name="point_cloud_topic" default="/center_pandar40p/pandar40p_driver/lidar_points" />
    <arg name="sbg_ins_pose_topic" default="/sirius/sensor/ins/global_pose" />
    <arg name="save_path_name" default="/home/jwz/my_files/debug_ws/result_data/" />
    <arg name="bag_prefix" default="2019-12-16-22-07-14" />

    <node pkg="rosbag_parse" type="rosbag_parse_node" name="rosbag_parse" output="screen">

        <param name="point_cloud_topic"  value="$(arg point_cloud_topic)" />
        <param name="sbg_ins_pose_topic" value="$(arg sbg_ins_pose_topic)"/>
        <param name="save_path_name" value="$(arg save_path_name)"/>
        <param name="bag_prefix" value="$(arg bag_prefix)"/>

    </node>
</launch>

